custom_components/elasticsearch/datastreams/index_template.json

```diff
@@ -147,6 +162,17 @@
                                                     "type": "keyword"
                                                 }
                                             }
+                                        },
+                                        "floor": {
```

Comment:
I'm torn about placing `floor` here, as it's technically an attribute of the `area`. I think it's easier to consume where you have it, but the pedant in me says it doesn't belong here.

Let's leave it the way you have it for now.

Reply:
Are you sure? I'd be happy to move it under the Area

Reply:
Let's move it under `area` for both `entity.area` and `device.area`, if you don't mind.